### PR TITLE
Event Day 3 - Pet Collars!

### PR DESCRIPTION
**Pull request by Cheridan**
AND THIS ONE IS EVEN FULLY COMPLETED.

Ordered by Thunder11!

Adds pet collars. You can put them on corgis and cats and stuff. You can also wear them as an accessory BUT WHY WOULD YOU, WEIRDO.

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // END_PREFERENCES
 
 // BEGIN_INCLUDE
-#include "_maps\tgstation2.dm"
+#include "_maps\ministation.dm"
```
Comment (Cheridan) [inline tgstation.dme]: oh damnit

**Comment by RandomMarine** [comment]
Should be able to pen a name onto the collar to rename whatever it's put on.

**Comment by Nienhaus** [comment]
Grayscale collars please.

File: code/modules/clothing/under/ties.dm [hunk 2]
```diff
@@ -277,3 +277,8 @@
 	name = "striped blue scarf"
 	icon_state = "stripedbluescarf"
 	item_color = "stripedbluescarf"
+
+/obj/item/clothing/tie/petcollar //don't really wear this though please c'mon seriously guys
+	name = "pet collar"
```
Comment (Iamgoofball) [inline code/modules/clothing/under/ties.dm]: >clothing 

Let's not even tempt the furries.

**Comment by Fox-McCloud** [comment]
"BUT WHY WOULD YOU, WEIRDO."

I take it you've never met a furry before? Hahah

Jokes aside, big :+1: on this PR.

**Comment by Thunder12345** [comment]
Cheri you're supposed to be able to name any generic pet the first time you put a collar on it too.

**Comment by vista-** [comment]
do wearing them make you VALID?

**Comment by Thunder12345** [comment]
Probably not.

**Comment by Durkel** [comment]
Great, now I can own lizard slaves just like my grand pappy.

**Comment by Chocobro** [comment]
>you can wear it

sweat

**Comment by vista-** [comment]
Requesting admin policy on "wearing collars and nonhuman status".

**Comment by dannno** [comment]
ballgag

**Comment by Miauw62** [comment]
RE-ADD BALLGAG WHEN

**Comment by Chocobro** [comment]
wait is it in the autodrobe
if so :+1:

**Comment by GunHog** [comment]
How do you get them? Do only pet owners (Heads) get them?

**Comment by SeikoTheSeal** [comment]
Can't attach them to kitty ears for maximum naked furry cat lady very sad/10.

**Comment by Thunder12345** [comment]
The idea is for station pets to start with them, pet crates from cargo to come with a collar for the pet, and crates of them orderable for giving to Ian's puppies. Though since they're human wearable, could put one in the autodrobe too.

**Comment by vista-** [comment]
my 2 cents: Let's not turn /tg/station 13 into Furry Station 69.

**Comment by Xhuis** [comment]
make it so if you equip it you take 200 brain damage

**Comment by GunHog** [comment]
It is just a cosmetic item. Grow up.

**Comment by Xhuis** [comment]
i don't even care what people do with this w/e
as long as you can rename the pets they're attached to i'm happy

**Comment by Thunder12345** [comment]
Nah, you're only meant to be able to rename generic pets, named station/crate pets can't be renamed, and in theory already come with a collar.

**Comment by Xhuis** [comment]
fair enough

**Comment by Xhuis** [comment]
nice

**Comment by francinum** [comment]
We all know what's coming.

**Pull request merged**